Jropp/jason-ropp-js-iobp
# PWA-Mini: Service-Worker

## What It Does

Adds a file caching service worker to the project. 

GOOGLE WANT GOOLGE WANT. So I added it to get my score up to 74! for a PWA.

I used the [workbox-webpack-plugin](https://www.npmjs.com/package/workbox-webpack-plugin) which was pretty "plugin"-and-play _pun groan_. It caches the main files on load and registers the service worker with the browser.

## How To Test

Load the page, you should see a console.log that workbox is caching.
Also if you check service workers in your dev tools, service-worker.js should show up.

## Notes/Caveats

Currently doing more digging to see if there is anything else I need to do here for now.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:


## Dependencies

- [ ] #141
